lib/ansible/modules/apt_key.py

```diff
@@ -90,8 +90,8 @@
 
     - name: somerepo | apt source
       ansible.builtin.apt_repository:
-      repo: "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/somerepo.asc] https://download.example.com/linux/ubuntu {{ ansible_distribution_release }} stable"
-      state: present
+        repo: "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/somerepo.asc] https://download.example.com/linux/ubuntu {{ ansible_distribution_release }} stable"
```

Comment:
This line is too long. Shortening the example will resolve the issue:

```suggestion
        repo: "deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/myrepo.asc] https://download.example.com/linux/ubuntu {{ ansible_distribution_release }} stable"
```